packages/vscode-extension/package.nls.json

```diff
@@ -292,6 +292,11 @@
     "teamstoolkit.localDebug.output.tunnel.installSuccessMessage": "ngrok is installed at %s.",
     "teamstoolkit.localDebug.output.tunnel.skipInstallMessage": "Skip checking and installing ngrok as user has specified ngrok path (%s).",
     "teamstoolkit.localDebug.output.tunnel.createDevTunnelMessage": "Dev tunnel tag: %s.",
+    "teamstoolkit.localDebug.output.tunnel.deleteDevTunnelMessage": "Deleted Dev tunnel '%s'.",
```

Comment:
Does "Dev tunnel"'s 'D' need to be capitalized here?

Reply:
Updated.